substrate/client/cli/src/params/shared_params.rs

```diff
@@ -33,10 +33,10 @@ pub struct SharedParams {
 
 	/// Specify the development chain.
 	///
-	/// This flag sets `--chain=dev`, `--force-authoring`, `--rpc-cors=all`,
-	/// `--alice`, and `--tmp` flags, unless explicitly overridden.
-	/// It also disables local peer discovery (see --no-mdns and --discover-local)
-	#[arg(long, conflicts_with_all = &["chain"])]
+	/// This flag sets `--chain=dev`, `--force-authoring`, `--rpc-cors=all`, `--alice`, and `--tmp`
+	/// flags, unless explicitly overridden. It also disables local peer discovery (see `--no-mdns`
+	/// and `--discover-local`).
```

Comment:
Should we mention manual seal here? (I know it does not enable it for every node, but I think it would be good to have this information in cli).

Reply:
Added a note here: [d6faae7](https://github.com/paritytech/polkadot-sdk/pull/6646/commits/d6faae78623e90ec34135283bbb9cb4f590c3d37).